cpp/tests/quantiles/quantile_test.cu

```diff
@@ -527,29 +558,6 @@ TYPED_TEST(QuantileTest, TestInterpolateExtremaLow)
     test(testdata::interpolate_extrema_low<TypeParam>());
 }
 
-TYPED_TEST(QuantileTest, TestSortedAscendingNullBefore)
```

Comment:
No longer necessary since sorting is factored out of quantiles and the `ordered_indices` functionality is tested elsewhere.